src/builder/meta.rs

```diff
@@ -153,17 +153,19 @@ impl ObjectMetaBuilder {
         resource: &T,
         app_name: &str,
         app_version: &str,
-        app_managed_by: &str,
-        app_component: &str,
-        role_name: &str,
+        operator_name: &str,
+        controller_name: &str,
+        app_role: &str,
+        app_role_group: &str,
```

Comment:
This triggers a cargo lint due to #491, not sure if we want to tackle that in this PR...

Reply:
I would combine both tickets into one PR because they create a breaking change at the same place. If we would merge this PR and not immediately the PR for #491 then we would force people to adapt their code twice. But I would also not implement #491 hastily without a thourough refinement.

Reply:
Updated to account for #491 being merged.